lib/cmd-mod.bash

```diff
@@ -14,6 +14,9 @@ error_code=0
 #
 for sub in $(find_module_roots "${FILES[@]}" | sort -u); do
 	pushd "${sub}" > /dev/null || exit 1
+	if [ "${use_path_prefix:-}" -eq 1 ] && [ "${cmd_cwd_arg:-}" ]; then
+		OPTIONS+=("${cmd_cwd_arg}=${sub}")
```

Comment:
Ok I've been thinking on this for a while now ... the thing that is holding up is the `=${sub}` ...

I don't like assuming that the target tool will understand `arg=val` (vs `arg val`) ... 

SOO, I'm wondering if the quick fix is to treat `cmd_cwd_arg` as a template ... ie.

For golangic-lint, the value would be:
```shell
cmd_cwd_arg="--path-prefix={{sub}}"
```
And our insertion script would know to replace `{{sub}}` with the value of `${sub}` ...

This would make it so we don't have to assume the format of argument+value ...